### PR TITLE
Add *Troubleshooting Auth Issues* section to docs

### PR DESCRIPTION
**Pull request by ndowmon**
We get enough requests from these integration configs that I think it's about time we start to standardize and publish troubleshooting tips.

Would love to hear feedback from CS / Solutions about where to put these troubleshooting tips. Are these appropriate in the graph projects themselves? If so, do they belong in `/docs/jupiterone.md`? If so, is this the right place? (After both the **In Azure** section and the **In JupiterOne** section)?

### CODE REVIEW

File: docs/jupiterone.md [hunk 1]
```diff
@@ -142,6 +142,27 @@ information:
...
+in your job log, check the following common configuration errors:
+
+- **Verify the Application (client) ID and Application (client) Secret:** Make
+  sure that you've verified the proper value for client ID and client secret.
+  The client secret has both a **Value** property and a **Secret ID** property.
+  The **Secret ID** is unused - make sure you haven't accidentally used the
+  **Secret ID** as the **Client ID**.
+- **Verify that you've enabled the proper API permissions:** Make sure the
+  required API permissions (described above) are enabled for the application.
+- **Verify that the API permissions have been grated as "Application" and not
```
Comment (kenanwarren) [inline docs/jupiterone.md:158]: grated -> granted

File: docs/jupiterone.md [hunk 2]
```diff
@@ -142,6 +142,27 @@ information:
 
 4. Click **Create Configuration** once all values are provided.
 
+### Troubleshooting Authentication Issues
```
Comment (a-u-h-g) [inline docs/jupiterone.md:145]:
```suggestion
### Troubleshooting
```

File: docs/jupiterone.md [hunk 3]
```diff
@@ -142,6 +142,27 @@ information:
...
+
```
Comment (a-u-h-g) [inline docs/jupiterone.md:146]:
```suggestion

#### Authentication Errors
```

**Review by a-u-h-g — APPROVED** [review]
some suggestions for future proofing this section

File: docs/jupiterone.md (hunk 3 above)
Comment (ndowmon) [reply docs/jupiterone.md:148]: Thanks, made these changes.

File: docs/jupiterone.md (hunk 1 above)
Comment (ndowmon) [reply docs/jupiterone.md]: nice catch ⚾

**Review by jayson-jensen-pro — APPROVED** [review]

**Pull request merged**